src/components/buttons/level-button.ts

```diff
@@ -45,6 +46,11 @@ export default class LevelBloonButton {
         this.btnSize = this.bloonSize;
         this.lockSize = canvas.height / 13;
         this.textFontSize = (this.size) / 6;
+        this.pulse = false;
+    }
+
+    setPulse(shouldPulse: boolean, isSpecial: boolean) {
```

Comment:
isSpecial is used here, do we still need to pass this?

Reply:
where are we using setPulse method?

Reply:
my bad. thanks for this. please check again @bernhardccodev